polarisJawsApp/Db/polarisjawset.db

```diff
@@ -54,11 +56,100 @@ record(transform, "$(P)POLJAWSET:VGAP:_FAN") {
   field(CLCL, "(A+((A-C)*((D-G)/(D-E))))") # Jaw set 2
   field(CLCM, "(A+((A-C)*((D-H)/(D-E))))") # Jaw set 3
   field(CLCN, "(A+((A-C)*((D-I)/(D-E))))") # Jaw set 4
+  field(CLCO, "(A+((A-C)*((D-J)/(D-E))))") # Jaw set 5
   
   field(OUTK, "$(P)MOT:JAWS1:VGAP:SP PP")
   field(OUTL, "$(P)MOT:JAWS2:VGAP:SP PP")
   field(OUTM, "$(P)MOT:JAWS3:VGAP:SP PP")
   field(OUTN, "$(P)MOT:JAWS4:VGAP:SP PP")
+  field(OUTO, "$(P)POLJAWSET:JAWS5:VGAP PP")
   
 }
 
+record(ao, "$(P)POLJAWSET:VCENT:SP") {
+  field(ASG, "$(ASG=DEFAULT)")
+  field(DESC, "Global Vertical Setpoint")
+  field(FLNK, "$(P)POLJAWSET:VCENT:_FAN")
+  field(PREC, "3")
+  field(EGU, "")
+}
+
+record(ao, "$(P)POLJAWSET:HCENT:SP") {
+  field(ASG, "$(ASG=DEFAULT)")
+  field(DESC, "Global Horizontal Setpoint")
+  field(FLNK, "$(P)POLJAWSET:HCENT:_FAN")
+  field(PREC, "3")
+  field(EGU, "")  
+}
+
+record(dfanout,"$(P)POLJAWSET:VCENT:_FAN") {
+  field(DOL, "$(P)POLJAWSET:VCENT:SP PP")
+  field(OMSL, "closed_loop")
+ 
+  field(OUTA, "$(P)MOT:JAWS1:VCENT:SP PP")
+  field(OUTB, "$(P)MOT:JAWS2:VCENT:SP PP")
+  field(OUTC, "$(P)MOT:JAWS3:VCENT:SP PP")
+  field(OUTD, "$(P)MOT:JAWS4:VCENT:SP PP")
+  field(OUTE, "$(P)POLJAWSET:JAWS5:VCENT PP")
```

Comment:
Can we add (brief) comments explaining why Jaws 5 is treated differently. I know and you know, but someone reading this code a year from now might not know why jaws 5 breaks pattern. (This applies throughout the file)